site/data/roadmap.yaml

```diff
@@ -31,6 +35,10 @@
   status: ⚡ In progress
   url: https://bestpractices.coreinfrastructure.org/en/projects/24?criteria_level=1
   year: 2024
+- item: Ensure all alerts have accurate examples
```

Comment:
Make this more generic? Something about documentation because I'm also tackling http refs, alert refs, cross links, and examples. (6119, 7100, 8262, 8189)

Reply:
That look better?